### PR TITLE
Unnecessary jQuery file removed and Finding clicked link text code up…

### PR DESCRIPTION
**Pull request by me-shaon**
I found that the jquery script is not needed, as it is only used in a 3 line code. Replaced that with vanilla javascript. Also re-factored the code of getting link Text.

### CODE REVIEW

File: js/content.js [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
 
-// get the link name from the context menu click
-// used delegate to retrive the values even after the dom loads
-$(document).on("contextmenu", "a", function(){ 
-	title = $(this).text().trim();
-});
+document.addEventListener("mousedown", function(event){
+    //catch the right click event
+    if(event.button == 2) { 
+        title = event.target.innerText;//get the clicked event text
```
Comment (ssi-anik) [inline js/content.js:6]: You need to trim the title text.

File: js/content.js [hunk 2]
```diff
@@ -1,17 +1,14 @@
...
-// broadcasted messages listener
 chrome.runtime.onMessage.addListener(function(request, sender, sendResponse) {
-	// check the message
-	if(request.message === "linqueue_get_anchor_title"){
-		// return response
-		var response = {'title': title};
-		sendResponse(response);
-	}
+    if(request.message === "linqueue_get_anchor_title") {
+        sendResponse({value: title});
```
Comment (ssi-anik) [inline js/content.js:12]: I can't see any variable named value, where the value will be replaced with the text "title",

File: js/background.js [hunk 3]
```diff
@@ -1,29 +1,19 @@
...
-	    	saveData(key, value)
-	    });
-	});
+	var message = { "message": "linqueue_get_anchor_title" };
+	var key = getCurrentTime();
+	var value = {};
+
+	chrome.tabs.sendMessage(tab.id, message, function(response) {
+		//Title text got from content script
+		var title = (response && response.value)? response.value : "Title not found";
```
Comment (ssi-anik) [inline js/background.js:9]: I think it's good if you stick to the property of an object, like response should have a reasonable property like "response.title" 
The property "value" can be anything. Please change here.

**Comment by ssi-anik** [comment]
The pull request is denied. Added comment on the lines where you need to change. Please change and make another pull request.
Thank you.

File: js/content.js (hunk 2 above)
Comment (me-shaon) [reply js/content.js:12]: It's an anonymous object. **value** is the key of this object which is retrieved at the response at background.js as **response.value**.

Comment (ssi-anik) [reply js/content.js:12]: Please change the value to 'title' as it's a reasonable property name.

**Pull request closed without merging**